src/deluge/dsp/filter/filter_set.cpp

```diff
@@ -198,10 +192,9 @@ int32_t FilterSet::setConfig(int32_t lpfFrequency, int32_t lpfResonance, bool do
 }
 
 void FilterSet::reset() {
-	hpladder.reset();
-	lpsvf.reset();
-	hpsvf.reset();
-	lpladder.reset();
-	noiseLastValue = 0;
+	hpfilter.ladder.reset();
+	lpfilter.svf.reset();
+	hpfilter.svf.reset();
+	lpfilter.ladder.reset();
```

Comment:
this looks a little suspect now, since the filters share storage.

Reply:
oh good point - I don't think it's necessary at all

Reply:
It is necessary but zeroing the storage works - the reset for each filter just sets the state to 0 anyway

Reply:
cool, I guess just leave a comment to the effect of "all zeros must represent a reset state" on the union? the rest of this PR looks fine.

Reply:
done, thanks